addons/frag/XEH_postInit.sqf

```diff
@@ -9,4 +9,8 @@ if(isServer) then {
     [QGVAR(frag_eh), { _this call FUNC(frago); }] call EFUNC(common,addEventHandler);
 };
 
-[FUNC(masterPFH), 0, []] call CBA_fnc_addPerFrameHandler;
\ No newline at end of file
+[FUNC(masterPFH), 0, []] call CBA_fnc_addPerFrameHandler;
+
+//Cache for ammo type configs
+GVAR(cacheRoundsTypesToTrack) = createLocation ["NameVillage", [-10000,-10000,-10000], 0, 0];
+GVAR(cacheRoundsTypesToTrack) setText QGVAR(cacheRoundsTypesToTrack);

```

Comment:
@NouberNou might know, I just borrowed this from ACRE / Carma

ref https://github.com/NouberNou/carma2/blob/master/rv/addons/lib/XEH_pre_init.sqf#L8-L9